config-cell-cleanup.html.md.erb

```diff
@@ -29,7 +29,7 @@ PCF provides the following options for scheduling the disk cleanup process on Di
 
 * **Never clean up the Cell Disk**: Pivotal does not recommend using this option for production environments.
 * **Routinely clean up the Cell Disk**: This option makes the cell schedule a disk cleanup whenever a container is created. Running the disk cleanup process this frequently can result in a negative impact on the cell performance.
-* **Clean up Disk space once a threshold is reached**: This option makes the cell schedule the disk cleanup process only when a configurable disk space threshold is reached or exceeded.
+* **Clean up Disk space once usage fills disk**: This option makes the cell schedule the disk cleanup process only when downloaded layers encroach on the space reserved for other jobs (configurable).
```

Comment:
I think the field name is a little bit misleading-- the disk isn't actually filled since there is still a reserve. What about "Clean up Disk space when usage reaches the reserve" or something like that? Also -- what do we mean by "downloaded layers" here?

Reply:
Downloaded layers are the file system components that we use for application containers (application droplets).

We will trigger garbage collections when we hit Total disk - reserve. The reserve should be a set aside portion of the available disk. We had difficulty finding a concise way to express this, if you have suggestions we're totally open.